requirements.txt

```diff
@@ -1,69 +1,44 @@
-alabaster==0.7.10
 alembic==0.9.9
 Babel==2.6.0
 bcrypt==3.1.4
 beautifulsoup4==4.6.0
 cffi>=1.7.0
 configobj==5.0.6
-cookies==2.2.1
-cov-core==1.15.0
-coverage==4.5.1
 croniter==0.3.24
 cryptography==2.4.2
 defusedxml==0.5.0
-docutils==0.14
 ecdsa==0.13
-enum34==1.1.6
 Flask==1.0.2
 Flask-Babel==0.11.2
 Flask-Migrate==2.1.1
 Flask-Script==2.0.6
 Flask-SQLAlchemy==2.3.2
-funcparserlib==0.3.6
-gunicorn==19.8.1
 huey==1.11.0
 idna==2.6
-imagesize==1.0.0
-ipaddress==1.0.22
-itsdangerous==0.24
 Jinja2==2.10
 ldap3==2.6
-lxml==4.2.1
+lxml==4.2.5
 Mako==1.0.7
-MarkupSafe==1.0
-mock==2.0.0
 netaddr==0.7.19
-nose==1.3.7
-numpy==1.14.3
 oauth2client>=2.0.1
 passlib==1.7.1
 Pillow==5.1.0
 psycopg2>=2.6
-py==1.5.3
 pyasn1==0.4.3
 pycparser==2.18
 PyJWT==1.6.4
 PyMySQL==0.8.1
-pymysql-sa==1.0
 pyOpenSSL==18.0.0
-pyparsing==2.2.0
 pyrad==2.1
-pytest==3.6.1
-pytest-cov==2.5.1
-pytest-runner==4.2
 python-dateutil==2.7.3
 python-gnupg==0.4.4
 pytz==2018.4
 pyusb==1.0.2
 PyYAML==5.1
 qrcode==6.0
-redis==2.10.6
```

Comment:
Ah I think we actually need redis to use the huey-based queue.
I didn't know, but apparently redis is only an extra requirement of huey :)

Reply:
so it doesn't get installed by huey?

Reply:
Wait - as it is the requirements, we will add this, even if it would get installed by huey. The requirements contain *all* packages.

Reply:
I *thought* redis is a direct dependency of huey. But as it turns out, it is technically only an optional dependency, so it doesn't get installed automatically when we install huey :)

Reply:
fixed in d4ae1646

Reply:
Thanks!